solr/solrj-streaming/src/java/org/apache/solr/client/solrj/io/stream/HashJoinStream.java

```diff
@@ -55,7 +55,7 @@ public class HashJoinStream extends TupleStream implements Expressible {
 
   protected Tuple workingFullTuple = null;
   protected String workingFullHash = null;
-  protected int workngHashSetIdx = 0;
+  protected int workingHashSetIdx = 0;
```

Comment:
this would be a good thing right?  that way they know to fix their bug right!   I don't really want to spend a ton of effort on this deprecation for some code that I don't know has many users or has a logical easy way to emit deprecations etc...    Unless we just deprecate in the java code?